inception-project/inception-external-recommender
# Bugfix for FlairTagger when using certain input formats.

**What's in the PR**
* In the old version of the script, I fixed the problem with whitespaces in a pretty hackish way. This commit cleans this up, which solves a bug that occured for me when using certain input formats, where my hack failed to work properly. 
* The old implementation also restricted the flair tagger to only tag tokens as defined by the whitespaces, this is now solved as well.

**How to test manually**
* e.g. Import a text as CAS XMI in Inception, observe that interpunctuation will not be part of the span tagged by the recommender.

**Automatic testing**
* see my last PR about not being able to get the testing environment running

**Documentation**
